raiden/transfer/state_change.py

```diff
@@ -252,12 +257,17 @@ def __ne__(self, other):
 class ContractReceiveChannelNewBalance(StateChange):
     """ A channel to which this node IS a participant had a deposit. """
 
-    def __init__(self, channel_identifier, participant_address: address, contract_balance: int):
-        if not isinstance(participant_address, address):
+    def __init__(
+            self,
+            channel_identifier,
+            participant_address: typing.address,
+            contract_balance: typing.block_number):
```

Comment:
humm, contract balance is not a block number, I will fix in an up coming pr

Reply:
argh copy pasta ... sorry